app/routes/users/components/UserSettings.tsx

```diff
@@ -58,6 +60,7 @@ const validate = createValidator({
   gender: [required()],
   email: [required(), isEmail()],
   allergies: [isValidAllergy()],
+  githubUsername: [isValidGithubUsername()],
```

Comment:
It's a _bit_ unnecessary to have this both in the frontend and backend, but not very important.